weblate/trans/models/suggestion.py

```diff
@@ -224,6 +224,7 @@ def get_checks(self):
         fake_unit = copy(self.unit)
         fake_unit.target = self.target
         fake_unit.state = STATE_TRANSLATED
+        fake_unit.suggestion_id = self.pk
```

Comment:
This would be a simpler solution and it would not need ad-hoc attribute:

```suggestion
        fake_unit.id = -self.pk
```